boards/lilygo/ttgo_t3/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+.. _lilygo_ttgo_t3:
+
+Liligo ttgo t3
```

Comment:
I know we've had the "naming" conversation before, and I tried to do some more research. I *think* the commonly accepted name for this board is in fact "TTGO-LoRa32" (or more specifically TTGO LoRa32 V2.1_1.6)
It might make sense to use ttgo_lora32 as a the base name, and even look at whether the various flavors listed [here](https://github.com/Xinyuan-LilyGo/TTGO-LoRa-Series) are effectively something that could be dealt with using [board revisions](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#multiple-board-revisions)?
Thoughts?

Reply:
Hi Kartben,
The name of the product depends on the shop where you buy it. The manufacturer has a webshop and also at least a shop on aliexpress and on taobao, maybe more or some resellers.
My board was a "Lilygo T3" or "TTGO T3".
Let's take the manufacturer webshop as a reference, they have a LoRa/GPS category:
https://www.lilygo.cc/collections/lora-or-gps
I'd say I have a LoRa32 V2.1_1.6. If I understand it right, the "2.1" is the version of the software and the "1.6" is the hardware revision.
The page https://github.com/Xinyuan-LilyGo/TTGO-LoRa-Series
lists the "TTGO LOAR Series" (typo I guess): V1.0, V1.2 (T-Fox), V1.6, V2.0.
I assume the V1.0 and V1.2/T-Fox are obsolete (the V1.0 is listed in the shop, but only the housing is available, the device is sold out), the V1.6 is the current model. The V2.0 might be a future version, but the README was last edited five years ago... so did the V2.0 ever go into production?
The new version of the TTGO LoRa32 V1.6 is the "T3S3":
https://www.lilygo.cc/products/t3s3-v1-0
Then there is the T-Beam series with the variants:
Meshtastic,  SoftRF, SUPREME(SoftRF), SUPREME(Meshtastic). 
For now I assume there are two hardware variants of the T-Beam: "Normal" and "Supreme"
Non-ESP products are:
T-Echo (nRF52) and T-Motion (STM32L072)

I assume we agree on the vendor prefix "lilygo"
I currently have the "LoRa32". I might order the "T-Beam (SoftRF)" and a "T3S3"
I can change the board names to whatever makes most sense.
**Please suggest names for the boards.**

As for board revisions: I can add them, but without the physical device it might be hard to do it right. Especially if I'm not sure it ever made it to the market (LoRa32 V2.0?)

Reply:
Hi Kartben,
how about:

vendor: lilygo (unchanged)
boards:
- ttgo_lora32 (rename ttgo_t3 -> ttgo_lora32)
- ttgo_tbeam (possibly two board revisions)
- ttgo_t3s3 
- ttgo_techo
- ttgo_tmotion
So far, I only have the ttgo_lora32 (current name: ttgo_t3) hw revision V1.6
I ordered a ttgo_tbeam, if time permits I might add that.

Reply:
Hi Kartben,
how about:

vendor: lilygo (unchanged)
boards:
- ttgo_lora32 (rename ttgo_t3 -> ttgo_lora32)
- ttgo_tbeam (possibly two board revisions)
- ttgo_t3s3 
- ttgo_techo
- ttgo_tmotion
So far, I only have the ttgo_lora32 (current name: ttgo_t3) hw revision V1.6
I ordered a ttgo_tbeam, if time permits I might add that.

Reply:
I changed it accordingly and added the t8c3 (other PR)